docs/src/wallet-guide/apps.md

```diff
@@ -71,4 +71,8 @@ __**DEVS**__: integrate Zelcore wallet into your products by implementing Sollet
 ### Zelcore Wallet Security
 Zelcore utilizes a username/password schema to derive private keys for all assets. Security layers include blockchain-based 2FA PIN, mobile device biometrics, and passphrases to allow users to set up as much or as little security as they like. All hashing functions done on-device, and no login/account info is stored nor transmitted off your device. The private keys only exist on-device while logged in, upon logging out there is no digital footprint left of your PKs.
 
+## Huobi Wallet
```

Comment:
put this section not at the last but at the alphabetical order by wallet name.

Reply:
Hello, I have been put in the right place according to your request